make/toolchain-android-aarch64.cmake

```diff
@@ -0,0 +1,97 @@
+# **********************************************************
+# Copyright (c) 2014-2016 Google, Inc.    All rights reserved.
```

Comment:
Generally we update the 2nd year to the current year.

Reply:
Done.